Simplify and add url (#14)
IsraelHikingMap/graphhopper-docker-image-push: README.md

Images can be found here:
https://hub.docker.com/r/israelhikingmap/graphhopper

I would like to first and foremost thank the [graphhopper](https://www.graphhopper.com/) team for their hard work and amazing product!
They are doing a great job and we are truly happy to help by contributing to thier code base like we had done in the past.
Graphhopper team has decided not to build a docker image and this repository is here to bridge that gap.
This repository is extremely simple, all it does is the following:
1. Every night at 1 AM it builds the latest code using Github actions from the [graphhopper repository](https://github.com/graphhopper/graphhopper) and uploads the image to docker hub with the `latest` tag
2. It checks if there's a new version tag, and if so builds it and upload it as well with the relevant tag

> 3. Adds a graphhopper.sh file for ease of use

Reviewer: ```suggestion
3. Adds a graphhopper.sh file for ease of use

```
Restore blank line after list

That's all.

Feel free to submit issues or pull requests if you would like to improve the code here

> 

Reviewer: ```suggestion

By default, the docker image runs the `server` command. A typical usage would specify the input file or url for the server. For example:  
`docker run israelhikingmap/graphhopper --url https://download.geofabrik.de/asia/israel-and-palestine-latest.osm.pbf`  
```

> In order to use this image there you can edit the following environment variables:

Reviewer: ```suggestion
This docker image uses the following default environment setting:
```

```
JAVA_OPTS: "-Xmx1g -Xms1g -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"
```

> You can also completely override the entry point and use this for example:

Reviewer: I suggest removing this paragraph. IMHO, overriding the entry point is rarely needed as all flags for `graphhopper.sh` can be supplied as arguments to the `docker run` command.

Images can be found here:
https://hub.docker.com/r/israelhikingmap/graphhopper

I would like to first and foremost thank the [graphhopper](https://www.graphhopper.com/) team for their hard work and amazing product!
They are doing a great job and we are truly happy to help by contributing to thier code base like we had done in the past.
Graphhopper team has decided not to build a docker image and this repository is here to bridge that gap.
This repository is extremely simple, all it does is the following:
1. Every night at 1 AM it builds the latest code using Github actions from the [graphhopper repository](https://github.com/graphhopper/graphhopper) and uploads the image to docker hub with the `latest` tag
2. It checks if there's a new version tag, and if so builds it and upload it as well with the relevant tag
3. Adds a graphhopper.sh file for ease of use

That's all.

Feel free to submit issues or pull requests if you would like to improve the code here

This docker image uses the following default environment setting:
```

> JAVA_OPTS: "-Xmx1g -Xms1g -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"

Reviewer: ```suggestion
JAVA_OPTS: "-Xmx1g -Xms1g"
```
